pkg/validation/exception/validate.go

```diff
@@ -5,10 +5,24 @@ import (
 
 	"github.com/go-logr/logr"
 	kyvernov2alpha1 "github.com/kyverno/kyverno/api/kyverno/v2alpha1"
+	"github.com/kyverno/kyverno/pkg/webhooks"
+)
+
+const (
+	namespacesDontMatch = "PolicyException resource namespace must match the defined namespace."
+	disabledPolex       = "PolicyException resources would not be processed until it is enabled."
```

Comment:
@chipzoller can you review the warning messages here ?

Reply:
This seems fine.

Reply:
Ok